### ISSUE TITLE
Move to Jazzband or other shared project space

### ISSUE THREAD

**Issue by j5awry** [noirbizarre/flask-restplus#726]
flask-restplus, even after adding multiple new maintainers, is continuing to fall behind requests. We are all donating our time and expertise, and there's still more work than available time.

We should think about moving to Jazzband or another shared project space. This gives us:

1. more possible maintainers and contributors
2. a good set of rules for adding maintainers and contributors

Jazzband may not be the right fit, but it's at least someplace to start

https://jazzband.co/about

**Comment by andreykurilin** [comment]
Also, https://opendev.org/ can be taken into account (the space created after managing the collaboration of such a huge system like OpenStack).

**Comment by SteadBytes** [comment]
This is an interesting idea, though I have no experience with anything of the sort.

Personally, I have been unable to put much time towards Flask-RESTPlus the last few weeks due to starting a new job - apologies for the absence. However things are settling down and I'm hoping to dedicate some more time to the project :+1: 

To be honest I think a good first step might be to just have a total clear out of old issues/PRs. If a PR is no longer applicable/old enough that it would be very difficult to understand the original context then it could be closed. Same goes for issues, as well as any issues that are not actual bugs/haven't been verified. If the issue/PR ought to be revisited we can encourage authors to update and open them again. It may also be worth closing feature requests that we don't intend to work on any time soon too. After a bit of 'housekeeping' and perhaps a decision on some more concrete future plans hopefully we'll be able to better direct our efforts. Any thoughts? :smile:

**Comment by rcpeters** [comment]
I would suggest merging some of the clearer and/or important PRs first to show the project has momentum and outside pull request are appreciated. Then when closing old PRs you can include a message explaining how many PRs were accepted.

**Comment by SteadBytes** [comment]
Yes I apologise if I wasn't clear. Recent/clear PRs can be merged if they are appropriate. Otherwise PRs can be closed along with a message to the author explaining the reason. Something along the lines of:
> This PR has been closed due to an organisational effort within the Flask-RESTPlus project. Your contribution *is* valued, however the burden of issues/PRs has gotten out of hand and the maintainers would like a clean start in order to continue development of the project. If you feel your issue/PR is still relevant please get in touch with the maintainers to create a new issue. For more information see https://github.com/noirbizarre/flask-restplus/issues/726

**Comment by rcpeters** [comment]
👍
